### PR TITLE
feat: Ad Targeting – Unsure

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?
Adds types around Targeting which we’re not sure about, which as per the JSDoc:

```ts
/**
 * I think all of these are deprecated?!?
 */
export type UnsureTargeting = { /* … */ }
```

### `getUnsureTargeting`

You get back what you pass to it, but I think it should all go, go, go away!

## Why?

JSDoc style means the documentation is as close as possible to the actual code.

### CODE REVIEW

File: src/targeting/unsure.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	/**
+	 * Kru**x** user segments – [see on Ad Manager][gam]
+	 *
+	 * * Type: _Dynamic_
+	 *
+	 * [gam]: https://admanager.google.com/59666047#inventory/custom_targeting/detail/custom_key_id=200247
+	 *
+	 * @deprecated We no longer use Krux
+	 */
+	x: string;
```
Comment (mxdvl) [inline src/targeting/unsure.ts:58]: @Amouzle confirmed this can be removed

File: src/targeting/unsure.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	 */
+	slot: string; // TODO: Narrow to known IDs
+};
+
+/* -- Methods -- */
+
+/* -- Targeting -- */
+
+export const getUnsureTargeting = (
+	targeting: UnsureTargeting,
```
Comment (zekehuntergreen) [inline src/targeting/unsure.ts:55]: Have we thought about creating a log when any of these properties are used in order to confirm whether or not they're deprecated?

Comment (mxdvl) [reply src/targeting/unsure.ts:55]: Yes: https://docs.google.com/spreadsheets/d/10vsny8P_hdtBSNBp3DSRpRV49e_mVJ24Q43NY4askeY/

Comment (Amouzle) [reply src/targeting/unsure.ts:55]: Happy with removing `gdrm` & `ms`. Just checking that `slot` will only be removed from the [BuildPageTargeting](https://github.com/guardian/frontend/blob/8bf15042f04c15875afbba5c5f93fcfa979e4357/static/src/javascripts/projects/common/modules/commercial/build-page-targeting.ts#L289)?

Comment (mxdvl) [reply src/targeting/unsure.ts:55]: Yes, only from the page targeting, not for individual slots.

**Comment by mxdvl** [comment]
Closing this in favour of the deprecating these values in frontend:
- `gdncrm` https://github.com/guardian/frontend/pull/24424
- `ms` TBA
- `slot` TBA

File: src/targeting/unsure.ts (hunk 1 above)
Comment (mxdvl) [reply src/targeting/unsure.ts]: https://github.com/guardian/frontend/pull/24431

**Pull request closed without merging**